assets/app/scripts/controllers/catalog/images.js

```diff
@@ -12,9 +12,26 @@ angular.module('openshiftConsole')
     $scope.projectImageRepos = {};
     $scope.openshiftImageRepos = {};
     $scope.builders = [];
-    $scope.images = [];
+    $scope.nonBuilders = [];
     $scope.sourceURL = $routeParams.builderfor;
 
+    var isBuilder = function(imageRepo, tag) {
+      if (!imageRepo.spec || !imageRepo.spec.tags) {
+        return false;
+      }
+
+      var i, imageTags, categoryTags;
```

Comment:
s/imageTags/imageTagSpec/

 Just trying to capture our understanding in variable names so the next time we look at this we won't lose our mind